.travis/lib-verify.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+verify::flake8() {
```

Comment:
Would it be better to move all the lib-*.sh functions into one file? I'm not sold on having this many files, especially when a few of them only have a single function in them.

Reply:
I just like splitting things up logically (I almost prefer smaller bash files anyway -- they're easier to manage)